src/platforms/android/manual-configuration.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: Manual Configuration
+sidebar_order: 2
+description: "Learn how to manually configure the Sentry Android SDK."
+---
+<PlatformContent includePath="getting-started-primer" />
+
+<PlatformSection noGuides notSupported={["android", "dart", "elixir", "flutter", "perl", "react-native", "unity", "unreal"]}>
+
+<Alert level="info" title="Using a framework?">
```

Comment:
I think this alert is also not being displayed:
![image](https://user-images.githubusercontent.com/4999776/218738389-a728b69d-62c2-4ace-bc43-8d8153fe7cf7.png)

maybe we could remove it